tridentctl/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM busybox
+
+ARG TRIDENT_VERSION="19.01.0"
+
+RUN set -ex && \
+    wget -q https://github.com/NetApp/trident/releases/download/v${TRIDENT_VERSION}/trident-installer-${TRIDENT_VERSION}.tar.gz && \
+    tar xzvf trident-installer-${TRIDENT_VERSION}.tar.gz && \
+    mv trident-installer/tridentctl /tridentctl && \
+    chmod +x /tridentctl
+
+RUN set -ex && \
+    wget -q https://storage.googleapis.com/kubernetes-release/release/$(wget -O- -q https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && \
+    mv kubectl /kubectl && \
+    chmod +x /kubectl && \
+    /kubectl version --client
+
+FROM debian
+
+LABEL io.whalebrew.name tridentctl
+LABEL io.whalebrew.config.environment '["KUBECONFIG","HOME"]'
```

Comment:
Sometimes I use kubeconfig file in the current directory. In kubectl, we can use `--kubeconfig` flag, but tridentctl does not have that flag. So in order to use any kubeconfig file, there is no choice but to use `KUBECONFIG` environment variable.

Reply:
yes this is exactly what I meant, in case $KUBECONFIG is in a different directory, there is no chance for it to be mounted in the container, or am I missing something?

Reply:
OK, I remove KUBECONFIG env val. If I need it in future, I'll reconsider how to use it.